backends/dpdk/DpdkXfail.cmake

```diff
@@ -96,6 +92,11 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-example-mirror-packet-error3.p4
   )
 
+p4c_add_xfail_reason("dpdk"
+  "declaration not found"
```

Comment:
why is this supposed to fail with p4c-dpdk only?

Reply:
it is failing may be due to some bug in compiler frontend. I am just recording this test for fixing it later.